src/RouteSwitch.js

```diff
@@ -58,13 +65,31 @@ function RouteSwitch() {
           }
         />
         <Route
-          path="/profile"
+          path="/account"
           element={
             <RequiresAuth>
-              <Profile />
+              <Account />
             </RequiresAuth>
           }
-        />
+        >
+          <Route index element={<Profile />} />
+          <Route
+            path="/account/profile"
+            element={
+              <RequiresAuth>
```

Comment:
Here you can use, <Outlet/> inside your RequiresAuth Wrapper, and make use of Nesting Route to like this:
```js
<Route element={<RequiresAuth/>}>
    <Route path="/account/addressess" element={<Addressess />} />
    <Route path="/account/addressess" element={<Profile/>} />
</Route>
````